recipes/cmake/3.x.x/conanfile.py

```diff
@@ -20,6 +20,7 @@ class CMakeConan(ConanFile):
         "with_openssl": "auto",
     }
 
+    short_paths = True
```

Comment:
Is this needed? We prefer to avoid the `short_paths` if it is not strictly needed.

Reply:
I see it failed in the build, ok. I'm triggering the job again to see if it works now.